crowbar_framework/app/models/ceph_service.rb

```diff
@@ -137,9 +137,9 @@ def create_proposal
     end
     mon_nodes = mon_nodes.take(mon_nodes.length > 2 ? 3 : 1)
 
-    mds_node = select_nodes_for_role(nodes, "ceph-mds", "controller").first
+    mds_node = select_nodes_for_role(nodes, "ceph-mds", "storage").first
 
-    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "controller").first
+    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "storage").first
```

Comment:
Let's also have in mind, this is just the automatic pre-deployment; user is still allowed to change it